Cargo.toml

```
@@ -221,6 +221,7 @@ pulsar = { version = "2.0.0", default-features = false, features = ["tokio-runti
 rand = { version = "0.8.3", default-features = false, features = ["small_rng"] }
 rand_distr = { version = "0.4.0", default-features = false }
 rdkafka = { version = "0.26.0", default-features = false, features = ["libz", "ssl", "zstd"], optional = true }
+redis = { version = "0.20.0", features = ["connection-manager", "tokio-comp", "tokio-native-tls-comp"], optional = true }
```

```suggestion
redis = { version = "0.20.0", default-features = false , features = ["connection-manager", "tokio-comp", "tokio-native-tls-comp"], optional = true }
```

We have started to include `default-features = false` more recently. That may mean you need to adjust the `features` to include needed features (I see the default featureset is `["acl", "streams", "geospatial", "script"]`).